applications/nrf5340_audio/src/drivers/cs47l63_reg_conf.h

```diff
@@ -31,9 +31,18 @@
 /* clang-format off */
 /* Set up clocks */
 const uint32_t clock_configuration[][2] = {
+#if CONFIG_AUDIO_SAMPLE_RATE_HZ == 16000
```

Comment:
Maybe better to use AUDIO_SAMPLE_RATE_16000_HZ etc. here?

Reply:
Fixed